cli/bpmetadata/types.go

```diff
@@ -18,18 +20,23 @@ type BlueprintMetadataSpec struct {
 
 // BlueprintInfo defines informational detail for the blueprint
 type BlueprintInfo struct {
-	Title         string
-	Source        *BlueprintRepoDetail
-	Version       string                 `json:",omitempty" yaml:",omitempty"`
-	ActuationTool BlueprintActuationTool `json:"actuationTool,omitempty" yaml:"actuationTool,omitempty"`
-	Description   *BlueprintDescription  `json:",omitempty" yaml:",omitempty"`
-	Icon          string                 `json:",omitempty" yaml:",omitempty"`
+	Title          string
+	Source         *BlueprintRepoDetail
+	Version        string                  `json:",omitempty" yaml:",omitempty"`
+	ActuationTool  BlueprintActuationTool  `json:"actuationTool,omitempty" yaml:"actuationTool,omitempty"`
+	Description    *BlueprintDescription   `json:",omitempty" yaml:",omitempty"`
+	Icon           string                  `json:",omitempty" yaml:",omitempty"`
+	DeploymentTime BlueprintDeploymentTime `json:"deploymentTime,omitempty" yaml:"deploymentTime,omitempty"`
```

Comment:
Should we call this `BlueprintTimeEstimate` to align better with `BlueprintCostEstimate` and since it will always vary?

Reply:
Yeah that's fine. I've prefixed the attributes within this to denote they are seconds i.e. `ConfigurationSecs` & `DeploymentSecs`.